java/gazelle/configure.go

```diff
@@ -55,13 +55,18 @@ func (jc *Configurer) KnownDirectives() []string {
 		javaconfig.ModuleGranularityDirective,
 		javaconfig.TestMode,
 		javaconfig.ExcludeArtifact,
+		javaconfig.TestFilePatterns,
```

Comment:
Can we sort this list?

Reply:
Done - also renamed all the constants to have a `Java` prefix so that the constants and the values sort the same.